src/lib.rs

```diff
@@ -87,6 +90,16 @@ pub struct Wizer {
     #[cfg_attr(feature = "structopt", structopt(long = "allow-wasi"))]
     allow_wasi: bool,
 
+    /// When using WASI during initialization, should `stdin`, `stderr`, and
+    /// `stdout` be inherited?
+    #[cfg_attr(feature = "structopt", structopt(long = "inherit-stdio"))]
+    inherit_stdio: Option<bool>,
```

Comment:
Instead of using `Option<bool>` here and then `unwrap_or` below, you could use `bool`, but set a `default_value` in the `cfg_attr`. That'd also improve the `--help` output.

Reply:
* `foo: bool` turns into whether `--foo` is passed or not, and
* `foo: Option<bool>` turns into `--foo true` or `--foo false` or nothing was passed

`structopt` doesn't allow you to use `default_value` with `foo: bool` because there is no way to specify false-ness in that case.

Altogether, this means that if you want something to be true by default, you have to use `Option<bool>`.

I htink adding a `default_value` would further improve the generated help text, though. I'll do that.

Reply:
Ah interesting, `structopt` doesn't allow `default_value` for `Option`s either.

I'll just add documentation by hand.